test/e2e-helm/e2e_helm_suite_test.go

```diff
@@ -55,33 +65,14 @@ var _ = BeforeSuite(func() {
 	By("preparing the prerequisites on cluster")
 	tc.InstallPrerequisites()
 
-	By("initializing a Helm project")
-	err = tc.Init(
-		"--plugins", "helm",
-		"--project-version", "3-alpha",
-		"--domain", tc.Domain)
-	Expect(err).NotTo(HaveOccurred())
-
 	By("using dev image for scorecard-test")
 	err = tc.ReplaceScorecardImagesForDev()
 	Expect(err).NotTo(HaveOccurred())
 
-	By("creating an API definition")
-	err = tc.CreateAPI(
-		"--group", tc.Group,
-		"--version", tc.Version,
-		"--kind", tc.Kind)
-	Expect(err).NotTo(HaveOccurred())
-
 	By("replacing project Dockerfile to use Helm base image with the dev tag")
 	err = testutils.ReplaceRegexInFile(filepath.Join(tc.Dir, "Dockerfile"), "quay.io/operator-framework/helm-operator:.*", "quay.io/operator-framework/helm-operator:dev")
 	Expect(err).Should(Succeed())
 
-	By("turning off interactive prompts for all generation tasks.")
-	replace := "operator-sdk generate kustomize manifests"
-	err = testutils.ReplaceInFile(filepath.Join(tc.Dir, "Makefile"), replace, replace+" --interactive=false")
-	Expect(err).Should(Succeed())
-
 	By("checking the kustomize setup")
 	err = tc.Make("kustomize")
 	Expect(err).NotTo(HaveOccurred())
```

Comment:
```suggestion
```
We can remove it as well.

Reply:
Technically, we should have never needed this since other targets that need `kustomize` would end up pulling it. However, I thought we explicitly added this to try and pull it early in the process since the `kustomize` download seems to flake fairly often.

Reply:
Yes, I think it was a copy-paste from go tests before of the fix https://github.com/operator-framework/operator-sdk/pull/4090

Reply:
Definitely could have been. I'm going to leave it in for now since it isn't strictly related to this PR. We can do a follow up to remove it from all e2e's at the same time if that's what we decide to do.